source/decisions/2025-11-15-trustee-quorum.rst

```diff
@@ -0,0 +1,14 @@
+2025-11-15 Quorum for Trustee Meetings
+======================================
+
+Resolution
+----------
+
+Background
+--------------------
```

Comment:
```suggestion
Background
----------

```

Reply:
Oops. Thanks. Too used to a linter catching these!